frontend/src/components/ThemedInput/ThemedTextArea.css

```diff
@@ -10,6 +10,9 @@
 	/* the textarea is automatically resized as the user types */
 	/* disable user resizing */
 	resize: none;
+	&:focus {
+		outline-offset: 0.1rem;
+	}
```

Comment:
Didn't know you could nest stuff like this! Pretty cool. However I'd be wary of browser support https://caniuse.com/css-nesting

for safety, I'd opt for a seperate block of css for the focus styles:

```css
.main-class {
// styles
}

.main-class :focus{
  outline-offset: 0.1rem;
}
```

Reply:
Oh wow, didn't know that had limited browser support, nice catch! Yep, will change that!